[2023-03-01 12:49:35,557] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-14T22:05:00+00:00 [queued]>
[2023-03-01 12:49:35,723] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-14T22:05:00+00:00 [queued]>
[2023-03-01 12:49:35,726] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:49:35,729] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 12:49:35,730] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:49:35,836] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-11-14T22:05:00+00:00
[2023-03-01 12:49:35,856] {standard_task_runner.py:52} INFO - Started process 13597 to run task
[2023-03-01 12:49:35,939] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-11-14T22:05:00+00:00', '--job-id', '3192', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmp2o68dqc6', '--error-file', '/tmp/tmp1nq03u9g']
[2023-03-01 12:49:35,949] {standard_task_runner.py:77} INFO - Job 3192: Subtask get_name
[2023-03-01 12:49:36,402] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-11-14T22:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 12:49:36,617] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-11-14T22:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-14T22:05:00+00:00
[2023-03-01 12:49:36,706] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 12:49:36,767] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221114T220500, start_date=20230301T124935, end_date=20230301T124936
[2023-03-01 12:49:37,119] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01 12:49:37,237] {local_task_job.py:146} INFO - Task exited with return code 0
